### ISSUE TITLE
many:many edge cannot be paginated

### ISSUE THREAD

**Issue by xixixao** [xixixao/convex-ents#25]
This
```ts
const result = await ctx
  .table("messages")
  .getX(messageId)
  .edge("tags")
  .paginate(paginationOpts);
```
should paginate the underlying edge table before loading the ents on the other end of the edge (tags in this example).

**Comment by xixixao** [comment]
Also: Make sure we can order by _creationTime from either side of the edge.

**Comment by xixixao** [comment]
Fixed in 1a8f7be
